tests/e2e.test.js

```diff
@@ -295,17 +296,34 @@ describe('Ordering', () => {
     }
   });
 
-  it.skip('orders grid items by color', async () => {
-    // Items are ordered using the CSS property `order`...
-    await expect(page).toClick('#order-color');
+  it('orders grid items by color', async () => {
+    const $items = await page.$$('li.grid-item');
+    for (let i = 0; i < $items.length; i++) {
+      const $button = await $items[i].$('button.grid-item__color');
 
-    const $gridItems = await page.$$('.copy-color');
-    for (let i = 0; i < $gridItems.length; i++) {
-      const $gridItem = $gridItems[i];
-      const hex = hexes[i];
-      await expect($gridItem).toMatch(`#${hex}`);
+      const $text = await $items[i].evaluate((el) =>
+        JSON.stringify(el.getAttribute('style')),
+      );
+      const $idx = parseInt($text.substring(15, $text.length - 1));
+
+      const $color = await $button.evaluate((el) =>
+        JSON.stringify(el.textContent),
+      );
+
+      await expect($color).toMatch(`#${hexes[$idx]}`);
```

Comment:
This test currently verifies whether the color of the card matches the color at the index specified by the CSS variable `--order-color` for that card in the `hexes` array. I don't think it verifies whether the ordering is displayed correctly as it tests wither `--order-color` is set correctly. Yes these are related, yes these may be better than no test (I personally disagree, as it gives a false sense that everything is okay), but it doesn't test whether the human viewing the screen will see the grid items sorted by color.

@jorgeamadosoria, the test wasn't skipped because I couldn't come up with some test. It's skipped because I couldn't come up with a test that verifies the color sorting works **end to end**. This is likely impossible (or at least very, very hard) due to the use of the `order var(--order-color)` approach and will likely need a fix related to #86 to be testable.

If you think this test is useful, I'm happy to approve it. I want to stress though that it is important to remember that it is not testing, **end to end**, whether or not color sorting is working.

Reply:
the test goes as far as it can. There is no way to test the way the page is going to look.
However, the css property uses order just fine, so checking that order is properly generated is enough. 
There is no other way of testing this. The DOM does not change. 
If you want to move the DOM around as discussed in #86, then this test can be amended, but as the coding stands right now, this is as good as it gets.

Reply:
This discussion is nearly identical to [this one](https://github.com/simple-icons/simple-icons-website/pull/110#discussion_r706673371), let's keep our focus there.